db/seeds.rb

```diff
@@ -8,6 +8,13 @@
 
 
 #Creates Questions in the database based on a category ID and a point value
+
+Game.delete_all
+Question.delete_all
+GameQuestion.delete_all
+UserGame.delete_all
```

Comment:
just to be clear, we are deleting user game histories here, is this just until release?

Reply:
@ILyell  what was the thought process behind deleting all the data in the seed file?